code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -246,3 +266,36 @@
 	var/obj/item/ammo_casing/ammo = ammo_type
 	caliber = initial(ammo.caliber)
 	..()
+
+/obj/item/weapon/gun/projectile/derringer
+	name = "derringer"
+	desc = "It's not size of your gun that matters, just the size of your load. Uses .357 rounds." //OHHH MYYY~
+	icon_state = "derringer"
+	item_state = "concealed"
+	w_class = 2
+	origin_tech = list(TECH_COMBAT = 2, TECH_MATERIAL = 2, TECH_ILLEGAL = 3)
+	handle_casings = CYCLE_CASINGS //player has to take the old casing out manually before reloading
+	load_method = SINGLE_CASING
+	max_shells = 2
+	ammo_type = /obj/item/ammo_casing/a357
+
+/obj/item/weapon/gun/projectile/luger
+	name = "\improper P08 Luger"
+	desc = "Not some cheap Scheisse .45 caliber Martian knockoff! This Luger is an authentic reproduction by RauMauser. Accuracy, easy handling, and its signature appearance make it popular among collectors... and space Nazis. Uses 9mm rounds."

```

Comment:
I can do that. It's a holdover from a story arc with Vorestation that involved killing space Nazis.